src/requirements.txt

```diff
@@ -4,7 +4,7 @@ Flask==1.1.2
 Flask-Cors==3.0.9
 monotonic==1.5
 pysftp==0.2.9
--e git+https://github.com/Amsterdam/GOB-Core.git@v0.17.11t#egg=gobcore
+-e git+https://github.com/Amsterdam/GOB-Core.git@v0.17.11y#egg=gobcore
```

Comment:
Updating the core version in GOB-Export doesn't change anything for the export or the enhanced view. The views are created by GOB-Upload and served by GOB-API. GOB-API doesn't really need an update when an enhanced view is changed, but it's probably good practice to update the core version there.